code-samples/serving/hello-world/helloworld-shell/Dockerfile

```diff
@@ -2,9 +2,6 @@
 # https://git.busybox.net/busybox/tree/networking/httpd.c
 FROM busybox
 
-# Serve from this directory
-WORKDIR /var/www
```

Comment:
is this the default?

Reply:
Yes, see [L6](https://github.com/knative/docs/blob/62e6f14ff3dce6c5e1ea5b3e30f705dde599fb1e/code-samples/serving/hello-world/helloworld-shell/Dockerfile#L6) on main branch